client/src/components/StatsList/StatButton/StatButton.scss

```diff
@@ -20,3 +21,11 @@ $stat-btn-text-color: $grey-100;
     background-color: shade($stat-btn-background-color, 20%);
   }
 }
```

Comment:
Ah, I missed a change needed here sorry!

Add `--background-color: #{$stat-btn-background-color};` at the top of `.stat-button`
Change `background-color` in `.stat-button` to `background-color: var(--background-color);`
Change `background-color` in `&:hover, &focus` to `--background-color: #{shade($stat-btn-background-color, 20%)};`

Reply:
Thanks! I've made that change and now it works as expected and I now follow what's going on ⭐

Reply:
These changes are in there so I've resolved it